hazelcast-sql-core/src/main/java/com/hazelcast/sql/impl/calcite/opt/physical/MapScanPhysicalRule.java

```diff
@@ -40,8 +40,8 @@
 
     private MapScanPhysicalRule() {
         super(
-            OptUtils.single(MapScanLogicalRel.class, HazelcastConventions.LOGICAL),
-            MapScanPhysicalRule.class.getSimpleName()
+                OptUtils.single(MapScanLogicalRel.class, HazelcastConventions.LOGICAL),
```

Comment:
Minor note: it seems that your IDEA adds indents automatically whenever the class is opened. It is better to disable this option because it makes PRs bigger and adds unnecessary entries to the file history.

Reply:
Fixed in https://github.com/hazelcast/hazelcast/pull/18013/commits/7c3b68005a67b13eb76bab0cb363271dfa466e45